docs/io.cozy.identities.md

```diff
@@ -21,14 +21,14 @@ unique by identifier **and** cozyMetadata.createdByApp
   "_id": "62e5d66d6e11d19992b7efce794263f0",
   "identifier": "example@email.com",
   "contact": {
-      "address": [
+      "addresses": [
```

Comment:
I'm not in favor of changing those keys. `contact` entry here, is the same as our `io.cozy.contacts` doctype. And in this doctype, we use the singular https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.contacts.md 

Can we revert those changes? 🙏 .